src/numerics/petsc_matrix.C

```diff
@@ -913,6 +913,19 @@ void PetscMatrix<T>::close ()
   LIBMESH_CHKERR(ierr);
 }
 
+template <typename T>
+void PetscMatrix<T>::flush ()
+{
+  semiparallel_only();
+
+  PetscErrorCode ierr=0;
+
+  ierr = MatAssemblyBegin (_mat, MAT_FLUSH_ASSEMBLY);
```

Comment:
Looks like this dates back to at least 1996 petsc/petsc@6d4a8577250, so we should be good introducing this without version guards.

Reply:
I was in primary school at that time.

Reply:
lol

Reply:
Did... did @fdkong just call us old? I think he just called us old.

Reply:
You need him to speak up, @pbauman?  Sad, but they do say the hearing is the first thing to go...

Reply:
I should probably replace my gravatar one of these days... the baby in that picture starts after-school robotics club next week.

Reply:
> Did... did @fdkong just call us old? I think he just called us old.

He's not wrong.

Reply:
Whatever, speak for yourself. :P Excuse me while I go get those kids off my lawn...